src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 14
+#define SOF_ABI_MINOR 15
```

Comment:
Should be 14, almost missed this !

Reply:
checkpatch complained that it needed an ABI bump, are you saying we don't?

Reply:
No bump needed, as we have open space in 14. checkpatch is really there to remind us to check the ABI.